server/src/sim/makeccocs.py

```diff
@@ -19,11 +19,12 @@
 
 ## Set defaults for testing makeccocs
 default_progname = 'SBCC'
-default_ccparams = [] # [0.9, 0.28, 154000.0, 1.0]
-default_ccplot = [] # [2e6, [0, []]]
-default_coparams = [] # [0.3, 0.5, 0.7, 0.9] 
```

Comment:
No that should be fine. I didn't know how they would come from the frontend but I assumed at least they wouldn't be floats, so the rest of the logic is based on that. I can simplify it later.